Mage/Task/BuiltIn/Deployment/Strategy/GitRebaseTask.php

```diff
@@ -53,8 +53,10 @@ public function run()
         $result = $this->runCommandRemote($command);
 
         if ($result === false) {
-            $repository = $this->getConfig()->deployment('repository');
-            if ($repository) {
+            $scmConfig = $this->getConfig()->deployment('scm');
```

Comment:
Don't you think it's gonna be a BC change? People who use Magallanes and update it, they'd all need to change their configs. I would leave it here as it was.